packages/patrol_cli/lib/src/ios/ios_test_backend.dart

```diff
@@ -227,8 +251,8 @@ class IOSTestBackend {
   }
 
   Future<String> _sdkVersion(bool real) async {
-    // See the versions yourself:
-    // $ xcodebuild -showsdks -json | jq ".[] | {sdkVersion, platform}"
+    // See the versions yourself: $ xcodebuild -showsdks -json | jq ".[] |
+    // {sdkVersion, platform}"
```

Comment:
You just made it hard to copy. Nothing wrong in long comments like that

Reply:
Right. FIxed as well.